Shared/Samples/Add ENC exchange set/README.md

```diff
@@ -0,0 +1,37 @@
+# Add ENC exchange set
+
+Display nautical charts per the ENC specification.
+
+![Image showing the add ENC exchange set app](add-enc-exchange-set.png)
+
+## Use case
+
+The [ENC specification](https://docs.iho.int/iho_pubs/standard/S-57Ed3.1/20ApB1.pdf) describes how hydrographic data should be displayed digitally.
+
+An ENC exchange set is a catalog of data files which can be loaded as cells. The cells contain information on how symbols should be displayed in relation to one another, so as to represent information such as depth and obstacles accurately.
+
+## How to use the sample
+
+Run the sample and view the ENC data. Pan and zoom around the map. Take note of the high level of detail in the data and the smooth rendering of the layer.
+
+## How it works
+
+1. Specify the path to a local CATALOG.031 file to create an `EncExchangeSet`.
+2. After loading the exchange set, get the `EncDataset` objects asynchronously.
+3. Create an `EncCell` for each dataset. Then create an `EncLayer` for each cell.
+4. Add the ENC layer to a map's operational layers collection to display it.
+
+## Relevant API
+
+* EncCell
```

Comment:
"ENC" in the API names should be capitalized. For example, [ENCCell](https://developers.arcgis.com/swift/api-reference/documentation/arcgis/enccell). Can you updated them throughout the README and metadata please?